src/tests/additionalTests.spec.js

```diff
@@ -0,0 +1,63 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { SORTING_OPTIONS_FUNCTIONS } = require('../pages/Inventory.page');
+
+test.describe('', () => {
+    test('Perform and verify sorting', async ({ loginPage, inventoryPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+        const options = Object.keys(SORTING_OPTIONS_FUNCTIONS);
+
+        for (let i = 0; i < options.length; i++) {
+            const option = options[i];
+
+            await inventoryPage.sortingBy(option);
+
+            const actualItems = await inventoryPage.getSortedItems();
+            const expectedItems = await inventoryPage.getSortedItems(option);
+
+            await expect(actualItems).toMatchObject(expectedItems);
+        }
+    });
+
+    test('Verify that added  products are displayed correctly', async ({ loginPage, inventoryPage, shopingCartPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+        await inventoryPage.addItemToCartById(0);
+        await inventoryPage.addItemToCartById(1);
+
+        const firsProductInfo = await inventoryPage.getItemInfoById(0);
```

Comment:
Now let's increase the complexity. We need to add random products (random number of products to the cart)